CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs

```diff
@@ -1348,6 +1352,33 @@ private void StopGame(PresetConfigV2 gamePreset)
         }
         #endregion
 
+        #region Game Resizable Window Payload
+        public async void StartResizableWindowPayload(PresetConfigV2 gamePreset, CancellationToken token)
+        {
+            try
+            {
+                // Check if the game is using Resizable Window settings
+                IGameSettingsUniversal _Settings = CurrentGameProperty._GameSettings.AsIGameSettingsUniversal();
```

Comment:
Since this is already invoked inside the `GetLaunchArguments()` method in line https://github.com/CollapseLauncher/Collapse/blob/60fc2abce3f2401421291b5f0e4579331493bc20/CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs#L1353 

Can we move the invoker for the this method (`StartResizeableWindowPayload()` in line 1272) inside the launch argument builder so the method is not run twice ?

Reply:
I don't think ``StartResizeableWindowPayload()`` will be related with ``GetLaunchArguments()`` since it's only for building the argument string and not with the game launching routine. I think we can use the one available in ``StartGame()`` method. So we will use the ``IGameSettingsUniversal _Settings`` from ``StartGame()`` and pass it to methods which needs it as an argument.

Reply:
Yeah that should work too :nodders: